src/otx/algo/action_classification/movinet.py

```diff
@@ -32,16 +38,41 @@ def __init__(
         metric: MetricCallable = MultiClassClsMetricCallable,
         torch_compile: bool = False,
     ) -> None:
-        config = read_mmconfig("movinet")
+        self.load_from = "https://github.com/Atze00/MoViNet-pytorch/blob/main/weights/modelA0_statedict_v3?raw=true"
         super().__init__(
             label_info=label_info,
-            config=config,
             optimizer=optimizer,
             scheduler=scheduler,
             metric=metric,
             torch_compile=torch_compile,
         )
 
+    def _create_model(self) -> nn.Module:
+        model = self._build_model(num_classes=self.label_info.num_classes)
+        model.init_weights()
+        self.classification_layers = self.get_classification_layers(prefix="model.")
+
+        if self.load_from is not None:
+            load_checkpoint(model, self.load_from, map_location="cpu")
+
+        return model
+
+    def _build_model(self, num_classes: int) -> nn.Module:
+        return MoViNetRecognizer(  # type: ignore[no-untyped-call]
+            backbone=MoViNetBackbone(),  # type: ignore[no-untyped-call]
```

Comment:
I'm just wondering, what is `no-untyped-call`?
Could you share the mypy error?

Reply:
I got `error: Call to untyped function "MoViNetRecognizer" in typed context  [no-untyped-call]`

Reply:
Why this error occurred for only `recognizer` and `backbone`? What is difference with `cls_head`?

Reply:
fixed at cd8caa14861e508be1508b7e3ec38f5ec48e64aa
Thank you so much!